bento.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: bento
+  version: "1.9.0"
+  epoch: 0
+  description: Bento is a high performance and resilient stream processor, able to connect various sources and sinks in a range of brokering patterns and perform hydration, enrichments, transformations and filters on payloads.
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 5e88f465240e6268343854377a3c5bfd9b9823e7
+      repository: https://github.com/warpstreamlabs/bento
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/ClickHouse/clickhouse-go/v2@v2.32.0
+        github.com/golang/glog@v1.2.4
+        github.com/opencontainers/runc@v1.1.14
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X main.Version=v${{package.version}}
+        -X main.DateBuilt=$(date -u -d "@$SOURCE_DATE_EPOCH" "+%Y-%m-%dT%H:%M:%SZ")
+        -X github.com/warpstreamlabs/bento/internal/cli.Version=v${{package.version}}
+        -X github.com/warpstreamlabs/bento/internal/cli.DateBuilt=$(date -u -d "@$SOURCE_DATE_EPOCH" "+%Y-%m-%dT%H:%M:%SZ")
+      packages: ./cmd/bento
+      output: bento
+      tags: timetzdata
+
+  - runs: |
+      cp ./config/docker.yaml ${{targets.contextdir}}/bento.yaml
+
+# https://github.com/warpstreamlabs/bento/blob/main/resources/docker/Dockerfile#L35
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Compatibility package to place binary in the location expected by bento"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}
+          ln -sf /usr/bin/bento ${{targets.contextdir}}/bento
+    test:
+      pipeline:
+        - runs: |
+            test "$(readlink /bento)" = "/usr/bin/bento"
+
+update:
+  enabled: true
+  github:
+    identifier: warpstreamlabs/bento
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - curl
+  pipeline:
+    - runs: |
+        set -o pipefail
+        bento --version | grep -E "${{package.version}}"
+        bento --help
+    - runs: |
+        curl -sLO https://raw.githubusercontent.com/warpstreamlabs/bento/refs/tags/v${{package.version}}/config/docker.yaml
+        mv docker.yaml config.yaml
+    - uses: test/daemon-check-output
+      with:
+        start: |
+          bento -c config.yaml
+        expected_output: |
+          Launching a Bento instance
+          Listening for HTTP requests at: http://0.0.0.0:4195
```

Comment:
in post: step can we check the port is up? (wait-for-it) if possible, send simple HTTP request with curl